setup.py

```diff
@@ -253,8 +255,8 @@ def install_html5(root="/", install_dir="/usr/share/xpra/www/", config_dir="/etc
                 shutil.copyfile(src, config_file)
                 os.chmod(config_file, 0o644)
                 #and create a symlink from `install_dir`:
-                #pointing to the config_file without the root:
-                config_file = os.path.join(config_dir, fname)
+                #pointing to the config_file:
+                config_file = os.path.join(cdir, fname)
```

Comment:
Hmm, I think that is going to break things.
We want to point to the location where things will be installed - ignoring the temporary root which is used by packaging tools. (`rpmbuild`'s buildroot)

Reply:
Would `rpmbuild` accept a relative link?  (cf. `os.path.relpath`)

Reply:
Interesting idea - it should work.
There's bound to be some surprises with it, but I can't think of any problems with it.